coderd/database/dbgen/dbgen.go

```diff
@@ -892,6 +892,16 @@ func CustomRole(t testing.TB, db database.Store, seed database.CustomRole) datab
 	return role
 }
 
+func Frobulator(t testing.TB, db database.Store, orig database.Frobulator) database.Frobulator {
+	frob, err := db.InsertFrobulator(genCtx, database.InsertFrobulatorParams{
+		UserID:      orig.UserID,
+		OrgID:       orig.OrgID,
+		ModelNumber: orig.ModelNumber,
```

Comment:
We always convert zero values to some random data in `dbgen`.
```suggestion
		UserID:      takeFirst(orig.UserID, uuid.New()),
		OrgID:       takeFirst(orig.OrgID, uuid.New()),
		ModelNumber: takeFirst(orig.ModelNumber, testutil.GetRandomName(t)),
```